doc/advanced-usage.md

```diff
@@ -598,4 +603,4 @@ if (_messages.where((e) => e.id == 'lastReadMessageId').isEmpty) {
     _chatKey.currentState?.scrollToUnreadHeader();
   });
 }
-```
```

Comment:
Why removibg the tics?

Reply:
Must have been removed unintentionally. The docs aren't updated as well, as it will be by them after discussion.